### ISSUE TITLE
Scan Inspector on 13" laptop screen

### ISSUE THREAD

**Issue by kleinann** [locdb/locdb-frend#436]
If the issue is complex, please try to divide it into smaller units and create one issue for each unit.

Your issue may already be reported!
Please search on the [issue track](../) before creating one.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Browser Name and version:
* Operating System and version (desktop or mobile):
* Link to your project:

**Comment by LauraErhard** [comment]
Earlier I tried this on a 13'' Laptop and I didn't have any issues. But I see that you used the Edge browser, maybe the issue is with the browser and not the size of your screen?!

**Comment by kleinann** [comment]
Yes, that's possible. I wasn't aware that we don't support Edge. Well, it's probably not that much in use in libraries anyway. We'll just have to document it then.
